lychee-bin/src/parse.rs

```diff
@@ -54,6 +54,13 @@ mod tests {
         let mut custom = HeaderMap::new();
         custom.insert(header::ACCEPT, "text/html".parse().unwrap());
         assert_eq!(parse_headers(&["accept=text/html"]).unwrap(), custom);
+
+        let mut custom_with_equals = HeaderMap::new();
+        custom_with_equals.insert("x-test", "check=this".parse().unwrap());
+        assert_eq!(
+            parse_headers(&["x-test=check=this"]).unwrap(),
+            custom_with_equals
+        );
```

Comment:
Can we put that into a separate test function? Makes it easier to troubleshoot each test individually in the future.

Reply:
I made the change. 😉